examples/src/main/java/com/mapbox/navigation/examples/core/ReplayHistoryActivity.kt

```diff
@@ -165,6 +181,61 @@ class ReplayHistoryActivity : AppCompatActivity() {
         )
     }
 
+    /** Rendering the set route event **/
+
+    private val options: MapboxRouteLineOptions by lazy {
+        MapboxRouteLineOptions.Builder(this)
+            .withRouteLineResources(
+                RouteLineResources.Builder()
+                    .routeLineColorResources(
+                        RouteLineColorResources.Builder().build()
+                    )
+                    .build()
+            )
+            .withRouteLineBelowLayerId("road-label")
+            .withVanishingRouteLineEnabled(true)
```

Comment:
The video in the PR doesn't show the route line vanishing. The example doesn't have all of the necessary configuration for it to work properly. I suggest setting this to false if you don't intend to use it in the example.

Reply:
Hmm I think this is because it's not map matching 🤔

Reply:
All our examples are broken with the map matcher. Check our the Routeline and Route Arrow Example

<img width="370" alt="Screen Shot 2021-07-19 at 3 26 23 PM" src="https://user-images.githubusercontent.com/3021882/126235248-b6aa863c-efc4-4be5-b567-86033e7dae68.png">

Reply:
https://github.com/mapbox/mapbox-navigation-android/issues/4640

Reply:
If you meant to use the vanishing route line in your example you're missing the `addOnIndicatorPositionChangedListener`. There maybe some other issue if as you say the other examples aren't working but your example specifically is missing that listener.

Reply:
The route line and arrow example is working (vanishing) for me on the main branch.

Reply:
Updated!